### ISSUE TITLE
Problem with C compiler and SonarLint.

### ISSUE THREAD

**Issue by mvp-2003** [microsoft/vscode-docs#6454]
So I had been running C programs quite well, but I, totally by mistake, moved the important files for "gcc" into some different folder, which I don't remember and I am unable to run C programs now. The VS Code terminal says "Path to shell executable "C:\User..." does not exist. Now I am not to great with computers, so I don't know how to fix this problem. This probably is not a VS Code issue but I am still writing it down here, expecting for some help. Thank You.

**Comment by gregvanl** [comment]
Hi @mvp-2003 Have you tried reinstalling gcc? For your initial setup, were you working through a tutorial or class instructions? If so, you could reference those to repair your compiler installation.

**Comment by mvp-2003** [comment]
I did not consider reinstalling gcc because at first I did not want to use up more space for the same program. Unfortunately, I am left with no other options other than re-installing gcc. I think that is the only solution I can have. And I actually did refer to a video for installing gcc because I myself did not know how to install it. Then I followed everything the instructor said in the videos, but I don't think I remember the instructions. I think it was a video on YouTube.
